src/vunnel/providers/amazon/parser.py

```diff
@@ -100,7 +100,7 @@ def _get_alas_html(self, alas_url, alas_file, skip_if_exists=True):
             return content  # noqa: RET504
 
         try:
-            self.logger.debug(f"downloading ALAS from {alas_url}")
+            self.logger.info(f"downloading ALAS from {alas_url}")
```

Comment:
why change this to info? Since this downloads a lot of info and can get rather chatty, this feels like a lower logging level.

Reply:
same comment in a few places

Reply:
based on an offline conversation -- ignore this